integration-tests/init/setversion.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const desc = Reflect.getOwnPropertyDescriptor(process.versions, 'node')
+
+desc.value = process.env.FAKE_VERSION
```

Comment:
Let's use a real old version instead. The version number is not what would typically cause issues, but rather things that we do that could be unsupported in older versions, and that includes the bailout logic.

Reply:
@rochdev Would one version prior to the current bottom of the engines field range suffice?

Reply:
To be honest I'd probably go with as low as we possibly can for the odd Node 4/6/8 that may try installing this, especially since it should be easy to support these versions for just the bailout script.

Reply:
New code looks good to me, but I'd say the minimum Node version should be lowered to at the very least 8, but if we can push 4 then that's even better. Both of those won't work on Apple Silicon though, so maybe the choice could be platform specific? In any case, this is not a blocker, but definitely something we should do, otherwise we have no idea if this works on older versions which may still fail.